src/LaunchDarkly.OpenFeature.ServerProvider/LaunchDarkly.OpenFeature.ServerProvider.csproj

```diff
@@ -39,7 +39,7 @@
 
   <ItemGroup>
     <PackageReference Include="LaunchDarkly.ServerSdk" Version="[6.3.2,7.0)" />
-    <PackageReference Include="OpenFeature" Version="0.5.0" />
+    <PackageReference Include="OpenFeature" Version="[1.0.0, 2.0.0)" />

```

Comment:
I'm not 100% sure but I think that this may end up resolving to the _lowest_ compatible version (1.0.0) rather than the highest 1.x version. The wildcard format (`1.*`) gives you the highest, and that's what we've been using in other .NET-based demo apps.
https://learn.microsoft.com/en-us/nuget/concepts/dependency-resolution
https://github.com/launchdarkly/hello-dotnet-server/blob/main/HelloDotNet/HelloDotNet.csproj

Reply:
Never mind, I was just confused because I'd been looking at a PR for a hello demo earlier so I had those in mind. I think the way you have the version range here is correct for the purposes of our library— it's up to the app to either pin to a particular OpenFeature version or specify a wildcard.

Reply:
I will keep that in mind for the hello app I am working on.